tests/test_tools/test_convert_dgv.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pandas as pd
+import pytest
+from unittest.mock import Mock, patch
+from tools.convert_dgv import main as convert_dgv_main
+import sys
+
+
+@pytest.mark.parametrize(
+    "filename,expected_file",
+    [
+        ["dgv_test.tab", "dgv_test_expected.tab"],
+    ],
+)
+def test_dgv_examples(tmp_path, filename, expected_file):
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+
+    output_path = str(tmp_path / "tmp_data.tab")
+    expected = pd.read_csv(os.path.join(data_dir, expected_file), sep="\t", header=0)
+    args = [
+        "python",
+        "--input",
+        os.path.join(data_dir, filename),
+        "--output",
+        output_path,
+    ]
+
+    with patch.object(convert_dgv_main, "main", Mock(), create=True):
```

Comment:
you shouldn't need to mock/patch main

Reply:
Removed in [960bd67](https://github.com/bcgsc/mavis/pull/336/commits/960bd67fdca774210a6cd5aac4c8c14bbe2a2a5f)